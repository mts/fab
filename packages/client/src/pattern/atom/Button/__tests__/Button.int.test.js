import { defaultRender } from './Button.int.render'

jest.mock('../../../../store/thunk/completeState', () => ({ setAppCompleteStateThunk: () => {} }))

describe('<Button />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
