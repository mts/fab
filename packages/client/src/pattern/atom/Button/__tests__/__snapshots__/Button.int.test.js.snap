// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Snaphot must match defaultRender 1`] = `
<button
  className="button"
  onClick={[Function]}
  type="button"
>
  Find
</button>
`;
